cfgov_setup.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from distutils.command.build_ext import build_ext as _build_ext
+from setuptools.command.bdist_egg import bdist_egg as _bdist_egg
+from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+
+from setuptools import Command
+
+from subprocess import call
+
+
+class FrontEndBuildFailure(Exception):
+    pass
+
+
+class build_frontend(Command):
+    """ A command class to run `setup.sh` """
+    description = 'build front-end JavaScript and CSS'
+    user_options = []
+    possible_scripts = ['frontendbuild.sh', 'setup.sh']
```

Comment:
I wonder if we should pick a default, and use an additional [keyword argument to setup](https://github.com/cfpb/complaint/pull/10/files#diff-2eeaed663bd0d25b7e608891384b7298R38) for something different.

So we'd default to setup.sh and then if it's different, do something like:

```
setup(
    ...
    do_frontend_build= True,
    frontend_build_script='notsetup.sh'
)
```